apps/src/code-studio/pd/international_opt_in/InternationalOptIn.jsx

```diff
@@ -8,6 +8,7 @@ import {DATE_FORMAT} from '../workshop_dashboard/workshopConstants';
 import {Row, Col, ControlLabel, FormGroup} from 'react-bootstrap'; // eslint-disable-line no-restricted-imports
 import i18n from '@cdo/locale';
 import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import _ from 'lodash';
```

Comment:
Please import only the functions you use in the file:
```suggestion
import {snakeCase} from 'lodash';
```